app/views/courses/_basic_details_tab.html.erb

```diff
@@ -120,7 +120,7 @@
           Locations
         </dt>
         <dd class="govuk-summary-list__value" data-qa="course__locations">
-          <% if course.sites.empty? %>
+          <% if course.sites.nil? || course.sites.empty? %>
```

Comment:
I still get an error from `CourseDecorator#has_site?`:

> undefined method `any?' for nil:NilClass

Presumably because the course's sites are `nil` and not `[]`. Should that also be protected from like it is here?